src/Features/ConfigPlus.cpp

```diff
@@ -377,6 +378,7 @@ static void FreeCondition(Condition *c) {
 		FreeCondition(c->binop_l);
 		FreeCondition(c->binop_r);
 		break;
+	case Condition::STEAMID:
```

Comment:
This causes a memory leak. You want to free `c->val` like in the other conditions.

Reply:
I would've put this up with the other conditions that use the same mechanism, but this works